src/SurveyService.Validation/UserAnswer/CreateUserAnswerRequestValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.UserAnswer;
+using LT.DigitalOffice.SurveyService.Validation.UserAnswer.Interfaces;
+
+namespace LT.DigitalOffice.SurveyService.Validation.UserAnswer;
+
+public class CreateUserAnswerRequestValidator : AbstractValidator<CreateUserAnswerRequest>, ICreateUserAnswerRequestValidator
+{
+  private readonly IOptionRepository _optionRepository;
```

Comment:
Remove this line, you use it only in the constructor

Reply:
Done